focuspoints.lrdevplugin/AppleDelegates.lua

```diff
@@ -0,0 +1,70 @@
+--[[
+  Copyright 2017 philmoz & Joshua Musselwhite, Whizzbang Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--]]
+
+--[[
+  A collection of delegate functions to be passed into the DefaultPointRenderer when
+  the camera is Apple (iPhone, iPad)
+--]]
+
+local LrStringUtils = import "LrStringUtils"
+require "Utils"
+
+AppleDelegates = {}
+
+--[[
+-- metaData - the metadata as read by exiftool
+--]]
+function AppleDelegates.getAfPoints(photo, metaData)
+
+  local imageWidth = ExifUtils.findFirstMatchingValue(metaData, { "Image Width", "Exif Image Width" })
```

Comment:
ExifUtils is loaded by DefaultPointRenderer, which is loaded by PointsRendererFactory before the other delegates are loaded.